horace_core/sqw/@sqw/func_eval.m

```diff
@@ -32,6 +33,13 @@
 %                - Most commonly just a numeric array of parameters
 %                - If a more general set of parameters is needed by the function, then
 %                  wrap as a cell array {pars, c1, c2, ...}
+%
+% Keyword Arguments:
+%   outfile     If present, the output of func_eval will be written to the file
```

Comment:
Would it be worth mentioning in the main part of the header that if an `outfile` is given the operation will be performed file-backed and therefore be slow?

Reply:
The `outfile` argument does not necessarily mean that the operation will be file-backed. If the input object is all in memory, then the operation will be performed in memory.

I think this does need to be cleaned up. Really, the `filebacked` kwarg only really needs to apply if the input is a file, so only needs to be parsed by the `algorithms/func_eval` overload. The `@sqw/func_eval` overload should just check if the input object's pixels are file-backed and act accordingly. This wouldn't be a change in behaviour, but would be a more sane way of handling the arguments.